jpyutil.py

```diff
@@ -38,8 +38,18 @@
 __version__ = "0.10.0.dev1"
 
 
-# Uncomment for debugging
-# logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.DEBUG)
+# Setup a dedicated logger for jpyutil.
+# This way importing jpyutil does not interfere with logging in other modules
+
+logger = logging.getLogger('jpyutil')
+# set to logging.DEBUG for debugging
+logger.setLevel(logging.INFO)
+
+ch = logging.StreamHandler()
+ch.setLevel(logging.DEBUG)
```

Comment:
Set level from environment variable `JPY_LOG_LEVEL`, default should be `INFO` as before. Document this please.